qb_dl.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Download helper for qutebrowser, uses wget
+
+
+__author__ = 'Michael Harig <floss@michaelharig.de>'
+__version__ = '0.1'
+__license__ = '''(c) 2022 MIT'''
+
+
+##### TODO
+# 
+# - Debug: After downloading, qb prints: ": no such command
+# - Test: What happens, when URL contains space(s)?
+# - DONE Function, that gets the content of the clipboard, preferably platform independent
+# - DONE Function, that spawns a wget -c, parameters: URL and filepath
+# - What did I mean with this? Function, that combines an optional parameter to the qb download command, or the basename of the link, and the configured/current download directory to a target filepath
+# - DONE Function, that executes the qb command
+# - progress indicator
+
+
+from tkinter import Tk      # needed for clipboard
+import subprocess as sp
+from os import environ
+from pathlib import Path
+import sys
+
+DLCMD = ['wget', '-q', '-c']
+
+
+def qexec(_cmd):
+    with open(environ['QUTE_FIFO'], 'w') as f:
+        f.write(_cmd)
+
+
+def getClipboard():
```

Comment:
why not get the url via argv or QUTE_URL?

Reply:
QUTE_URL does not work in any case, e.g. hints do not work when links are displayed by hover functionality.